x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/engine/engine_analytics.tsx

```diff
@@ -0,0 +1,206 @@
+                    {
+                      defaultMessage:
+                        'Gain insight into query frequency, how many queries returned no results, and how many clicks they have generated.',
+                    }
+                  )}
+                </EuiText>
+              </EuiFlexItem>
+              <EuiFlexItem grow={false}>
+                <EuiButtonEmptyTo
+                  iconType="eye"
+                  onClick={() => {
+                    KibanaLogic.values.navigateToUrl(ANALYTICS_PLUGIN.URL, {
+                      shouldNotCreateHref: true,
+                    });
+                  }}
+                  to={''}
+                >
+                  {i18n.translate(
+                    'xpack.enterpriseSearch.content.engine.overview.analytics.button.buttonLabel.viewAnalytics',
+                    {
+                      defaultMessage: 'View Analytics',
+                    }
+                  )}
+                </EuiButtonEmptyTo>
+              </EuiFlexItem>
+            </EuiFlexGroup>
+          </EuiFlexItem>
+        </EuiFlexGroup>
+      </EuiFlexGroup>
+      <div hidden>
+        {metricAttributesQueries ? (
+          <EngineAnalyticsLens attributes={metricAttributesQueries} metricAttributesQueriesFlag />
+        ) : null}
+        {metricAttributesNoResults ? (
+          <EngineAnalyticsLens
+            attributes={metricAttributesNoResults}
+            metricAttributesNoResultsFlag
+          />
+        ) : null}
+      </div>
+      <EuiPanel hasBorder>
+        <EuiFlexGroup wrap>
+          <EuiFlexItem grow>
+            <EuiText size="s">
+              <h2>
+                <FormattedMessage
+                  id="xpack.enterpriseSearch.content.engine.overview.analytics.chart.title"
+                  defaultMessage="{title}"
+                  values={{
+                    title: !isNoResultsCardVisible ? 'Queries - Total' : 'Queries - No Results',
+                  }}
+                />
+              </h2>
+            </EuiText>
+          </EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <EuiSuperDatePicker
+              start={startDate}
+              end={endDate}
+              onTimeChange={setTimeChange}
+              showUpdateButton={false}
+              width={'auto'}
+              aria-label={i18n.translate(
+                'xpack.enterpriseSearch.content.engine.overview.analytics.date.filters.endDateAriaLabel',
+                { defaultMessage: 'Filter by date' }
+              )}
+            />
+          </EuiFlexItem>
+        </EuiFlexGroup>
+
+        {/* Chart */}
+
+        <EuiFlexGroup wrap>
+          {xyAttributes ? <EngineAnalyticsLens attributes={xyAttributes} /> : null}
+        </EuiFlexGroup>
+        {/* Cards */}
+        <EuiFlexGroup wrap>
+          <EngineAnalyticsCard
+            queries={queriesCount}
+            percentage={queriesCountPercentage}
+            cardTitle="Total queries"
```

Comment:
suggestion: Pass a localised string here. I see you are trying to localise it within the component